util/find-doc-nits

```diff
@@ -864,19 +864,15 @@ if ( $opt_c ) {
 }
 
 if ( $opt_l ) {
-    foreach (@ARGV ? @ARGV : (glob('doc/*/*.pod'), glob('doc/*/*.pod.in'),
-                              glob('doc/internal/*/*.pod'))) {
+    foreach ( glob('doc/*/*.pod doc/internal/*/*.pod') ) {
         collectnames($_);
```

Comment:
Uhmmmmm...  why?  Also `doc/man7/openssl_user_macros.pod.in`

Reply:
We should run find-doc-nits after the source has been generated, not on both foo.pod and foo.pod.in